include/Vector.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+#include <iostream>
+#include <algorithm>
+
+class Vector {
+public:
+    Vector() = default;
```

Comment:
Are you sure you want your class to have a default constructor without any args? Maybe it's better in this case to give the other constructor's params default values instead